test/vstest.console.UnitTests/ExecutorUnitTests.cs

```diff
@@ -298,6 +300,47 @@ public void ExecutorShouldShowRightErrorMessage()
         }
     }
 
+    [TestMethod]
+    public void ExecutorShouldPrintsWarningIfRunningEmulatedOnARM64()
+    {
+        var mockOutput = new MockOutput();
+        Mock<IProcessHelper> processHelper = new();
+        processHelper.Setup(x => x.GetCurrentProcessArchitecture()).Returns(PlatformArchitecture.X64);
+        processHelper.Setup(x => x.GetCurrentProcessId()).Returns(0);
+        processHelper.Setup(x => x.GetCurrentProcessFileName()).Returns(@"c:\vstest.console.exe");
```

Comment:
I would use `X:` just to make it "clear" that we don't `vstest.console.exe` to exist at this location.

```suggestion
        processHelper.Setup(x => x.GetCurrentProcessFileName()).Returns(@"X:\vstest.console.exe");
```

Reply:
mmm you can have `x:` drive or?